common/lib/driver-definitions/src/test/types/validateDriverDefinitionsPrevious.ts

```diff
@@ -372,6 +372,7 @@ declare function get_current_InterfaceDeclaration_IDocumentStorageService():
 declare function use_old_InterfaceDeclaration_IDocumentStorageService(
     use: TypeOnly<old.IDocumentStorageService>);
 use_old_InterfaceDeclaration_IDocumentStorageService(
+    // @ts-expect-error compatibility expected to be broken
```

Comment:
I'm not an expert here, but I though an appropriate change in package.json should not require this change - please ping Helio for clarification

Reply:
currently the change in the package.json is expected to produce this change as part of the `typetests:gen` script so this is right